README.md

```diff
@@ -1,6 +1,8 @@
 ### Mautic Sparkpost Plugin
 
-This plugin enable Mautic 5 to run Sparkpost as a transport.
+This plugin enable Mautic 5 to run Sparkpost as an email transport. Features:
+- API transport. This transport can send up to 2000 emails per API request which makes it very fast compared to SMTP.
+- Bounce webhook handling. This plugin will unsubscribe contacts in Mautic based on the hard bounces while Sparkpost will take care of the soft bounce retrieals.
 
 #### Usage
 `composer require ts-navghane/sparkpost-plugin`
```

Comment:
```suggestion
```
The old repo `ts-navghane/sparkpost-plugin` is public archived and no longer available under https://packagist.org.
Can we please also remove this step? Thank you.

Reply:
I know. I'm trying to figure out how to create it under the Acquia namespace. But we can remove it till it's done.